packages/react-components/react-combobox/src/components/Combobox/useComboboxStyles.styles.ts

```diff
@@ -212,6 +213,9 @@ const useIconStyles = makeStyles({
       display: 'block',
     },
   },
+  hidden: {
+    display: 'none',
```

Comment:
is `display: 'none'` necessary? you already have logic to conditionally set the slot to `undefined` in the state hook.

I think it would be cleaner to either use css or js for both of these cases

Reply:
I use CSS and state. There are not rendering conditions in render functions anymore, based on @smhigley feedback

Reply:
Ah sorry, it also can't be `display: none`, since that also will remove it from the accessibility tree. We have a few instances of visually hidden but accessible content styles in [useAriaLiveStyles](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-toast/src/components/AriaLive/useAriaLiveStyles.styles.ts#L14-L20) or [useCardStyles](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-card/src/components/Card/useCardStyles.styles.ts#L347-L353)